src/agent/debuglet.ts

```diff
@@ -114,6 +160,7 @@ export class Debuglet extends EventEmitter {
   private controller_: Controller;
   private completedBreakpointMap_: {[key: string]: boolean};
 
+  private cachedPromise: CachedPromise;
```

Comment:
Just initialize here instead of from the constructor.

Reply:
Done.